proto/axelar/evm/v1beta1/service.proto

```diff
@@ -185,11 +185,21 @@ service QueryService {
 
   // ERC20Tokens queries the ERC20 tokens registered for a chain
   rpc ERC20Tokens(ERC20TokensRequest) returns (ERC20TokensResponse) {
-    option (google.api.http).get = "/axelar/evm/v1beta1/erc20_tokens/{chain}";
+    option (google.api.http) = {
+      get : "/axelar/evm/v1beta1/erc20_tokens/{chain}"
+      additional_bindings : {
+        get : "/axelar/evm/v1beta1/erc20_tokens/{chain}/type/{type}"
+      }
+    };
   }
 
   // TokenInfo queries the token info for a registered ERC20 Token
   rpc TokenInfo(TokenInfoRequest) returns (TokenInfoResponse) {
-    option (google.api.http).get = "/axelar/evm/v1beta1/token_info/{chain}";
+    option (google.api.http) = {
+      get : "/axelar/evm/v1beta1/token_info/{chain}/asset/{asset}"
```

Comment:
`asset` and `symbol` here should definitely use query params instead of path params.